<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>piworkorder</class>
 <widget class="QWidget" name="piworkorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3235</width>
    <height>1507</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>250</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#piworkorder{
background-color: rgb(255, 255, 255);
border: solid thin rgb(0, 0, 0);
}

/*QLabel{
	font: 10pt &quot;Verdana&quot;;
}

 QGroupBox::title{
	font: 10pt &quot;Verdana&quot;;
}*/
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="margin">
      <number>10</number>
     </property>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Juegos a Retirar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="nroLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fecha de Entrega:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="fechaValueLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hora Aprox.:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="horaValueLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Recibido:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="recibidoLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Trabajo:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string>A Nombre de:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="worknameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>62</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="barcodeLabel">
         <property name="font">
          <font>
           <family>Code 128</family>
           <pointsize>200</pointsize>
          </font>
         </property>
         <property name="text">
          <string>001-000001-00000001</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Como Realizar</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Completo:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="completeLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Como:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="howToLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Anillado:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="anilladoLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Anillar en:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="anillarWhereLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Costo</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Costo Fotocopia:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="fotocopiaValueLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Costo Anillado:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="anilladoValueLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Seña:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="sennaValueLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Total a Pagar:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="totalValueLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>62</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="readableBarcodeLabel">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>32</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Paginas</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="listView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>72</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Observaciones</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>72</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
